unstructured/ingest/evaluate.py

```diff
@@ -224,12 +225,15 @@ def _listdir_recursive(dir: str):
     return listdir
 
 
-def _write_to_file(dir: str, filename: str, rows: List[Any], headers: List[Any]):
+def _write_to_file(dir: str, filename: str, rows: List[Any], headers: List[Any], mode: str = "w"):
+    if mode not in ["w", "a"]:
+        raise ValueError("Mode not supported. Mode must be one of [w, a].")
     if dir and not os.path.exists(dir):
         os.makedirs(dir)
-    with open(os.path.join(os.path.join(dir, filename)), "w", newline="") as tsv:
+    with open(os.path.join(os.path.join(dir, filename)), mode, newline="") as tsv:
         writer = csv.writer(tsv, delimiter="\t")
-        writer.writerow(headers)
+        if mode == "w":
```

Comment:
is there an upcoming use case for mode a? this looks fine to add, but just wondering how it will be used

Reply:
I'm adding just in case we want to keep track of the eval output in the future :)